src/ReorderEligibility/ReorderEligibilityCheckerResponse.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\CustomerReorderPlugin\ReorderEligibility;
+
+class ReorderEligibilityCheckerResponse
```

Comment:
I think we've misunderstood about this class a little bit. What I was thinking about was:
1. Class `EligibilityCheckerResult` which contains some message what has gone wrong (and maybe a status? success/failure?)
2. Each checkers returns array/collection of `EligibilityCheckerResult`s 
3. Composite checker merges all responses into returns them to the controller
4. We need **one** more service that would convert each *result* into the flash message 

And that's all. I didn't dig deep into these `ReorderEligibilityCheckerResponseProcessor`s but I think they're just not needed. I think we should discuss it in live conversation.